packages/ai/core/tool/mcp/mcp-sse-transport.ts

```diff
@@ -16,9 +16,9 @@ export class SSEClientTransport implements MCPTransport {
     close: () => void;
   };
 
-  onClose?: () => void;
-  onError?: (error: unknown) => void;
-  onMessage?: (message: JSONRPCMessage) => void;
+  onclose?: () => void;
```

Comment:
Can we rename the class to `SSEMCPTransport` (I don't like long caps sequence but it's important to have consistent patterns imo)

Reply:
Similar to the above, the reason I wanted to keep `Client` in the name is to ensure users are aware this isn't a server transport!

Reply:
Hm - I don't think we'll add MCP servers anytime soon, so I'm inclined to vote for simplifying and not aligning to the original MCP lib on this (I don't want to much of their decisions bleed into the AI SDK, the lowercase callbacks were an exception)

Reply:
Noted!